go-controller/pkg/ovn/ovn.go

```diff
@@ -260,16 +260,27 @@ func (oc *Controller) WatchNodes() error {
 			if err != nil {
 				logrus.Errorf("error creating subnet for node %s: %v", node.Name, err)
 			}
+			err = oc.syncNodeManagementPort(node)
+			if err != nil {
+				logrus.Errorf("error creating Node Management Port for node %s: %v", node.Name, err)
+				return
```

Comment:
We should not be returning here. We should log the error and continue. If we return from here, then we will miss handling LB rules for that node. So, please remove the `return` statement. We do the same thing at L261.